CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_sap_source v0.1.1
+## Bugfix
+- Corrects the documentation regarding which schema the models are written to.
+
```

Comment:
No need for a CHANGELOG entry.

Reply:
@fivetran-joemarkiewicz 
Yeah I was wondering about this. Should we keep this note to include in the next release? And if so, should we add a placeholder for the next version. Maybe like v0.1.x. But it's not a significant change so I'm fine removing it entirely.

Reply:
Since this is a fairly small change we can omit the changelog entry.